packages/client/src/components/TransactionList.js

```diff
@@ -190,17 +112,27 @@ function TransactionList({ safeData, transactions = [{ foo: "bar" }] }) {
     );
   };
 
-  return transactions.map((transaction, index) => {
-    return (
+  const transactionsToShow = [];
+  transactions.forEach(tx => {
+    // only show successes / rejections
+    const isTransfer = tx?.blockEventData?.actionView?.type.includes('Transfer');
+    const isRejection = tx?.flowEventId.includes('ActionDestroyed');
```

Comment:
currently the design only shows successful transfers or rejected actions so filtering out anything else